### PR TITLE
Add on row click callbacks for table sortable

### LINKED ISSUE #1363: OnClick Event for Table Row in Table-Sortable
It would be very helpful to have a onClick Event for table Row in the Table-Sortable

### PR DESCRIPTION
**Pull request by LucaMele**
fixes #1363

### CODE REVIEW

File: src/components/30-organisms/table-sortable/index.js [hunk 1]
```diff
@@ -175,6 +183,31 @@ class AXATableSortable extends LitElement {
     return super.shouldUpdate(...args);
   }
 
+  handleOnclick = (ev, index, type) => {
```
Comment (danyball) [inline src/components/30-organisms/table-sortable/index.js]: CamelCase is missing here, should be `handleOnClick`

File: src/components/30-organisms/table-sortable/story.js [hunk 2]
```diff
@@ -69,6 +70,26 @@ storiesOf('Organisms/Table Sortable', module)
...
+      Inner Text is: ${JSON.stringify(textArray)}`;
+    };
+    const template = html`
+      <axa-table-sortable
+        onclick="onCallbackClick(arguments[0])"
+        innerscroll="500"
+        model="${JSON.stringify(model)}"
+      >
+      </axa-table-sortable>
+      <div id="renderArea"></div>
```
Comment (danyball) [inline src/components/30-organisms/table-sortable/story.js:88]: With my 13.3 Mac i havent seen this area. Should be better at the top of the table.

File: src/components/30-organisms/table-sortable/index.js [hunk 3]
```diff
@@ -175,6 +183,31 @@ class AXATableSortable extends LitElement {
...
+    this.onClick(details);
+    fireCustomEvent('click', details, this, { bubbles: false });
+  };
+
+  // if is focused, let the row be pressed via keyboard
+  onKeyPress = (...args) => {
+    const {
+      0: { charCode, code },
+    } = args;
+    if (~KEY_CODES.indexOf(charCode) || ~KEY_NAMES.indexOf(code)) {
```
Comment (danyball) [inline src/components/30-organisms/table-sortable/index.js]: `~` :D geil aber auch WTF. Kannte ich so noch nicht. Besser lesbar finde ich !== -1

File: src/components/30-organisms/table-sortable/index.js (hunk 1 above)
Comment (LucaMele) [reply src/components/30-organisms/table-sortable/index.js]: Thanks, will fix it

File: src/components/30-organisms/table-sortable/story.js (hunk 2 above)
Comment (LucaMele) [reply src/components/30-organisms/table-sortable/story.js:88]: Good idea! Thank you

File: src/components/30-organisms/table-sortable/index.js (hunk 3 above)
Comment (LucaMele) [reply src/components/30-organisms/table-sortable/index.js]: Is a bad habit of me. I will remove it :) agree is not readable and not good practice

File: src/components/30-organisms/table-sortable/story.js (hunk 2 above)
Comment (LucaMele) [reply src/components/30-organisms/table-sortable/story.js:88]: Ok, @danyball i cannot put it before as the help text would be shown before the table which would be much worst readable for our users

File: src/components/30-organisms/table-sortable/index.js (hunk 1 above)
Comment (LucaMele) [reply src/components/30-organisms/table-sortable/index.js]: fixed

File: src/components/30-organisms/table-sortable/index.js (hunk 3 above)
Comment (LucaMele) [reply src/components/30-organisms/table-sortable/index.js]: fixed

File: src/components/30-organisms/table-sortable/story.js (hunk 2 above)
Comment (danyball) [reply src/components/30-organisms/table-sortable/story.js:88]: When i start clicking at the demo i wonder that nothing happens. But thats not the main use-case i think.

**Review by danyball — APPROVED** [review]

**Pull request merged**